[NbConvertApp] WARNING | Config option `template_path` not recognized by `NotebookExporter`.
[NbConvertApp] Converting notebook analysis_notebook.ipynb to notebook
[NbConvertApp] Executing notebook with kernel: python3
[NbConvertApp] ERROR | Error while converting 'analysis_notebook.ipynb'
Traceback (most recent call last):
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/nbconvertapp.py", line 407, in export_single_notebook
    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/exporters/exporter.py", line 178, in from_filename
    return self.from_file(f, resources=resources, **kw)
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/exporters/exporter.py", line 196, in from_file
    return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/exporters/notebook.py", line 32, in from_notebook_node
    nb_copy, resources = super(NotebookExporter, self).from_notebook_node(nb, resources, **kw)
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/exporters/exporter.py", line 138, in from_notebook_node
    nb_copy, resources = self._preprocess(nb_copy, resources)
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/exporters/exporter.py", line 315, in _preprocess
    nbc, resc = preprocessor(nbc, resc)
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/preprocessors/base.py", line 47, in __call__
    return self.preprocess(nb, resources)
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/preprocessors/execute.py", line 361, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/preprocessors/base.py", line 69, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/nbconvert/preprocessors/execute.py", line 385, in preprocess_cell
    raise CellExecutionError.from_cell_and_msg(cell, out)
nbconvert.preprocessors.execute.CellExecutionError: An error occurred while executing the following cell:
------------------
diffselprefix = os.path.join(diffseldir, 'summary_')
groups = diffselbatch['group'].unique()

for seltype in ['mutdiffsel', 'positivesitediffsel']:
    print("\n{0} correlations:".format(seltype))
    plots = []
    for g in groups:
        plot = diffselprefix + g + '-' + seltype + 'corr.pdf'
        plots.append(plot)
    showPDF(plots) 
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mAssertionError[0m                            Traceback (most recent call last)
[0;32m<ipython-input-17-9db1efee262f>[0m in [0;36m<module>[0;34m[0m
[1;32m      8[0m         [0mplot[0m [0;34m=[0m [0mdiffselprefix[0m [0;34m+[0m [0mg[0m [0;34m+[0m [0;34m'-'[0m [0;34m+[0m [0mseltype[0m [0;34m+[0m [0;34m'corr.pdf'[0m[0;34m[0m[0;34m[0m[0m
[1;32m      9[0m         [0mplots[0m[0;34m.[0m[0mappend[0m[0;34m([0m[0mplot[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 10[0;31m     [0mshowPDF[0m[0;34m([0m[0mplots[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m/fh/fast/bloom_j/software/conda/envs/BloomLab_v2/lib/python3.6/site-packages/dms_tools2/ipython_utils.py[0m in [0;36mshowPDF[0;34m(pdfs, width)[0m
[1;32m     32[0m         [0mpdfs[0m [0;34m=[0m [0;34m[[0m[0mpdfs[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[1;32m     33[0m     assert all(map(os.path.isfile, pdfs)), "Can not find PDFs:\n{0}".format(
[0;32m---> 34[0;31m             '\n'.join(pdfs))
[0m[1;32m     35[0m     [0;32mtry[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m     36[0m         x = subprocess.check_output(['convert', '--version'], 

[0;31mAssertionError[0m: Can not find PDFs:
./results/diffsel/summary_2B06-mutdiffselcorr.pdf
AssertionError: Can not find PDFs:
./results/diffsel/summary_2B06-mutdiffselcorr.pdf

[NbConvertApp] Converting notebook analysis_notebook.ipynb to markdown
[NbConvertApp] Support files will be in analysis_notebook_files/
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Making directory results/analysis_notebook_files
[NbConvertApp] Writing 18141 bytes to results/analysis_notebook.md
